src/browser_launchers/playwright_launcher.js

```diff
@@ -74,17 +18,38 @@ export function apifyOptionsToLaunchOptions(launchContext) {
  *   Example: `http://bob:pass123@proxy.example.com:1234`.
  * @property {boolean} [useChrome=false]
  *   If `true` and `executablePath` is not set,
- *   Puppeteer will launch full Google Chrome browser available on the machine
+ *   Playwright will launch full Google Chrome browser available on the machine
  *   rather than the bundled Chromium. The path to Chrome executable
  *   is taken from the `APIFY_CHROME_EXECUTABLE_PATH` environment variable if provided,
  *   or defaults to the typical Google Chrome executable location specific for the operating system.
  *   By default, this option is `false`.
- * @property {(string|Object)} [launcher]
+ * @property {Object} [launcher]
  *   By default this function uses require("playwright").chromium`.
  *   If you want to use a different browser you can pass it by this property as `require("playwright").firefox
  */
 
-/**
+export class PlaywrightLauncher extends BrowserLauncher {
+    static optionsShape = {
+        ...BrowserLauncher.optionsShape,
+        launcher: ow.optional.object,
+    }
+
+    /**
+    * @param {PlaywrightLaunchContext} launchContext
+    * All `PlaywrightLauncher` parameters are passed via an launchContext object.
+    */
+    constructor(launchContext = {}) {
+        ow(launchContext, 'PlaywrightLauncherOptions', ow.object.exactShape(PlaywrightLauncher.optionsShape));
+
+        launchContext.launcher = launchContext.launcher || require('playwright').chromium; // eslint-disable-line
```

Comment:
why not default arguments like we use everywhere?

Reply:
Also I think we should keep the custom require functions. They had nice error messages that explained how to fix missing dependencies. It's especially useful for telling users that they might be using a wrong Docker image.